shared/src/packages/users/libs/validation-schemas/user-business-sign-up.validation-schema.ts

```diff
@@ -0,0 +1,32 @@
+import joi from 'joi';
+
+import { type UserBusinessSignUpRequestDto } from '../types/types.js';
+import { Regexp as REGEXP } from './enums/regexp.enum.js';
+import { UserValidationMessage } from './enums/user-validation-message.enum.js';
+import { userSignUpRules } from './user-sign-up.validation-schema.js';
+
+const companyNamePattern = new RegExp(REGEXP.COMPANY_NAME);
+const taxNumberPattern = new RegExp(REGEXP.TAX_NUMBER);
```

Comment:
I don't like that for each pattern you are writing a separate variable in this file. Maybe it's better to create some helper for wrapping to new RegExp(expression)

Reply:
something like `createRegexep=(regexp)=>new RegExp(regexp)`? Or I can use `password: joi.string().trim().required().pattern(new RegExp(Rule.PASSWORD)).messages({
    'string.empty': Message.REQUIRED,
    'string.pattern.base': Message.PASSWORD_NOT_VALID,
  }),`  I added variables at top of the file just for code readability. I'm not sure what approach is better